### ISSUE TITLE
Container element is empty - WYSIWYG content

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9675]
# Motivation

Siteimprove is reporting "Container element is empty". This is a WCAG 2.1 A compliance issue related to empty elements in the WYSIWYG editor.

When viewing the source in the WYSIWYG, the empty elements are not visible (examples of both `<ul></ul>` and `<ol></ol>`.

In order to fix manually, you likely need to delete line breaks while viewing source and then save to remove the empty elements.

Although this is fixable by a content person, it is not straight forward since the empty elements are not output in the source html within the WYSIWYG.

Seems like at a minimum, we should document this.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r68

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/998305/A11y/Page?pageId=40238913731&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/dWEyWHpON0M=;MDE0YTMyM2E=
- https://my2.siteimprove.com/Inspector/1002207/A11y/Page?pageId=47390882608&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/ZFVRWitIV3A=;YjJlMTYzMzc=
- https://my2.siteimprove.com/Inspector/989984/A11y/Page?pageId=42359354355&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/RDYwSlhsTnk=;MDFlOGUyOTc=

Example
- https://international.uiowa.edu/about
- https://stanleymuseum.uiowa.edu/people/annette-niebuhr
- https://tippie.uiowa.edu/content/tippies-laws

**Container element is empty**
Issue
1.3.1: Info and Relationships
Some roles are designed to contain other roles. This element has this type of role, but it doesn't contain any [required-owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain).
